astro/src/content/docs/lifecycle/authenticate-users/identity-providers/index.mdx

```diff
@@ -53,6 +53,25 @@ The following providers are available:
 
 If you're looking for a provider that is not listed here, review the open features in [GitHub](https://github.com/FusionAuth/fusionauth-issues/issues) and either vote or comment on an existing feature, or open a new feature request if you do not find an existing feature open.
 
+## Identity Provider Scope
+
+<Aside type="version">
+In version 1.62.0 and later FusionAuth supports creating tenant-scoped Identity Provider instances.
+</Aside>
+
+A tenant-scoped identity provider is an instance of an identity provider that is only usable in authentication workflows for the specified tenant. This can be useful to scope a particular identity provider configuration to users within a specific tenant.
+
+An identity provider instance that is not scoped to a specific tenant is global and available to all tenants. All identity provider instances created prior to version 1.62.0 are global.
+
+### Global Identity Providers and Tenant-scoped API Keys
+A tenant-scoped API key cannot make changes that could be seen or otherwise affect other tenants than the one it is scoped to. Because global identity providers span all tenants, you cannot use a tenant-scoped API key with a global identity provider via the [Identity Provider API](/docs/apis/identity-providers). You can, however, use a tenant-scoped API key to interact with things like identity provider links, as these are confined to a user within a tenant.
```

Comment:
```suggestion
A tenant-scoped API key can only alter its associated tenant. Because global identity providers span all tenants, you cannot use a tenant-scoped API key with a global identity provider via the [Identity Provider API](/docs/apis/identity-providers). You can, however, use a tenant-scoped API key to interact with things like identity provider links, as these are confined to a user within a tenant.
```
[suggestion]: this first sentence is complex enough to be a bit confusing. any chance we could trim the verbosity and negatives?